examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<p>If possible could you remove this output by running on a machine with a GPU?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. I had to uninstall TF 2.7.0 and install TF 2.3.0 for the sake of compatibility with libcudart.so.10.1</p>

Reply:
No, we don't support TF 2.3, only TF >= 2.6.1. I still see 2.7.0 in the notebook and this output:

Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /opt/intel/compilers_and_libraries_2018.1.163/linux/tbb/lib/intel64_lin/gcc4.7:/opt/intel/compilers_and_libraries_2018.1.163/linux/compiler/lib/intel64_lin:/opt/intel/compilers_and_libraries_2018.1.163/linux/mkl/lib/intel64_lin::/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64/:/opt/gurobi902/linux64/lib:/opt/gurobi902/linux64/lib
2022-02-02 08:15:05.744441: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

This probably means that you did not install cuda 11 and cudnn 8 in your conda environment, otherwise this library would be loaded. Could you run this notebook again with a clean conda environment in which you have done 
```
conda install -c conda-forge cudatoolkit=11.2 cudnn=8.1
```

Reply:
From other messages later on, it is shown that the code uses cudnn 7. It should use version 8 instead.

Reply:
For my Linux (x86_64, Ubuntu - 18.04) I could not install cudatoolkit=11.2 using the steps mentioned [here](https://developer.nvidia.com/cuda-11.2.0-download-archive?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1804&target_type=debnetwork) or using `conda install -c conda-forge cudatoolkit=11.2 cudnn=8.1`

Reply:
I think you could try removing these messages by updating LD_LIBRARY_PATH to include the location where libcudart.so.11.0 is. It may be under /anaconda/pkgs. If it does not exist anywhere, or if it is corrupt, you could try reinstalling cudatoolkit 11 or try a new VM.

Reply:
@aeroabir this could be to add `LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/path/to/libcudart.so.11.0` when activating the conda environment. For that edit the file `$ENV/etc/conda/activate.d/env_vars.sh` which exports the variables every time you activate the environment. 
A quicker fix is just to erase the logs of that cell :-)

Reply:
I've replicated this type of message on an (AzureML) VM. It seems to appear on TF 2.7, not on 2.6.
It is actually rather strange because I get the message even after I add the directory where the library is in the LD_LIBRARY_PATH.
Also I noticed that the message disappears after rerunning the cell.
I just pushed a cleaner version of the notebook.